common/thorhelper/thorsort.cpp

```diff
@@ -197,27 +198,27 @@ inline void * * mergePartitionsRev(const ICompare & compare, void * * result, si
     ret2 += (n2-1);
     while (n--)
     {
-       if (compare.docompare(*ret1, *ret2) >= 0)
+       if (compare.docompare(*ret1, *ret2) <= 0)
```

Comment:
Previously this was confusing - the arguments were reversed from mergePartitions.  This reverses the parameters so the function is more consistent with mergePartitions()